### DISCUSSION TITLE
device list add topsec

### DISCUSSION THREAD

**Discussion by yearcode** [ktbyers/netmiko#3488] [General]
hi ktbyers:
      We have Topsec(brand) firewall, only 2 device,  would add topsec to netmiko device list? 
thanks!

**Comment by ktbyers** [comment]
Someone would need to write the Netmiko driver for it and perform the standard Netmiko `show` and `config` tests on this driver, but assuming that is done, I should be able to review it and incorporate it into Netmiko.
